packages/sdk-react/src/components/receivables/InvoicesTable/useInvoiceRowActionMenuCell.test.tsx

```diff
@@ -0,0 +1,258 @@
+import { receivableListFixture } from '@/mocks/receivables';
+import {
+  checkPermissionQueriesLoaded,
+  createRenderWithClient,
+  testQueryClient,
+} from '@/utils/test-utils';
+import {
+  ActionEnum,
+  PermissionEnum,
+  ReceivablesStatusEnum,
+} from '@monite/sdk-api';
+import { renderHook, waitFor } from '@testing-library/react';
+
+import {
+  filterInvoiceActionMenuAllowedItems,
+  InvoicesTableRowAction,
+  useInvoiceRowActionMenuCell,
+} from './useInvoiceRowActionMenuCell';
+
+describe('useInvoiceRowActionMenuCell', () => {
+  test('returns default action menu items', async () => {
+    const { result } = renderHook(
+      () =>
+        useInvoiceRowActionMenuCell({
+          onRowActionClick: jest.fn(),
+        }),
+      {
+        wrapper: createRenderWithClient(),
+      }
+    );
+
+    await waitFor(() => checkPermissionQueriesLoaded(testQueryClient));
```

Comment:
Let's remove `checkPermissions*` check from the code. It's an outdated approach. Please use `await waitUntilTableIsLoaded()`

Reply:
No, I don't think so. In my opinion - `waitUntilTableIsLoaded(...)` is a semantic garbage. The idea is that a table spinner just isn't enough, since actions depend on the permissions load state. Unfortunately, I haven't found any way to otherwise ensure that permissions have been fully loaded for _this case_.

Reply:
Removed, since the component logic was completely changed.